crates/enarx-config/src/lib.rs

```diff
@@ -15,6 +22,9 @@ fn default_addr() -> String {
 }
 
 #[derive(Clone, Debug, PartialEq, Eq)]
+/// Name of a file descriptor
+///
+/// This is used to export a list of file descriptor names in the `FD_NAMES` environment variable.
```

Comment:
This lacks context on what `FD_NAMES` actually looks like

Reply:
fixed